fem/bilinearform.cpp

```diff
@@ -1763,12 +1763,24 @@ void MixedBilinearForm::FormRectangularSystemMatrix(
 
    mat->Finalize();
 
-   if (test_P) // TODO: Must actually check for trial_P too
+   if (test_P && trial_P)
    {
       SparseMatrix *m = RAP(*test_P, *mat, *trial_P);
       delete mat;
       mat = m;
    }
+   else if (test_P)
+   {
+      SparseMatrix *m = mfem::Mult(*test_P, *mat);
```

Comment:
This should be `TransposeMult`, shouldn't it?

Reply:
Good catch, yes it should be.